### ISSUE TITLE
AVT 1 - React Datatable w/batch actions has DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3741]
Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP IBM Accessibility July 2019 Ruleset

Detailed Description
Go to the React Datatable Component w/ batch actions
Run DAP and the following issue is found on the copy button (see screenshot):
<img width="1189" alt="Screen Shot 2019-08-15 at 12 49 31 PM" src="https://user-images.githubusercontent.com/21676914/63114866-9426a300-bf5b-11e9-837d-206ee8b153f0.png">
<img width="1190" alt="Screen Shot 2019-08-15 at 12 50 11 PM" src="https://user-images.githubusercontent.com/21676914/63114875-9852c080-bf5b-11e9-9c8b-a9940b96196e.png">
Open the menu in the toolbar and one additional violation is reported.
<img width="1241" alt="Screen Shot 2019-08-15 at 2 44 59 PM" src="https://user-images.githubusercontent.com/21676914/63121962-4ca81300-bf6b-11e9-933f-118e785a34cb.png">

Note: This issue has 8 violations that are the same as issue #3740 with 1 additional violation, "input label references must be valid."

**Comment by fbarroso24** [comment]
The following issue, https://github.ibm.com/IBMa/accessibility-web-engine/issues/214, was recently opened for the `DAP` team to investigate to see if the 1st 2 violations `Each form control must have an associated label` and `Provide descriptive text in LABEL elements` are False-Positive violations.

**Comment by asudoh** [comment]
No DAP error in the issue description is reproducible with latest `master` and November 2019 ruleset. The link @fbarroso24 provided (thanks!) confirms that.
